pkg/api/dashboard.go

```diff
@@ -292,6 +299,27 @@ func (hs *HTTPServer) PostDashboard(c *models.ReqContext, cmd models.SaveDashboa
 
 	dashSvc := dashboards.NewService(hs.SQLStore)
 	dashboard, err := dashSvc.SaveDashboard(dashItem, allowUiUpdate)
+
+	// Tell everyone listening that the dashboard changed
+	if hs.Live.IsEnabled() {
+		if dashboard == nil {
+			dashboard = dash // the original request
+		}
+		channel := hs.Live.GrafanaScope.Dashboards
+		liveerr := channel.DashboardSaved(c.SignedInUser.ToUserDisplayDTO(), cmd.Message, dashboard, err)
```

Comment:
Can't understand this - we are trying to call `channel.DashboardSaved` but not checking `err` from `dashSvc.SaveDashboard`. Is it ok?

Reply:
yes -- it actually the heart of the change.  This will broad broadcast the change and return a `202` rather than an error when a gitops observer is listening